datacenter/dtr/2.2/guides/admin/configure/external-storage/index.md

```diff
@@ -6,116 +6,72 @@ title: Configure DTR image storage
 ---
 
 After installing Docker Trusted Registry, one of your first tasks is to
-designate and configure the Trusted Registry storage backend. This document provides the following:
+designate and configure the Trusted Registry storage backend.  This document
+provides the following:
 
 * Information describing your storage backend options.
 * Configuration steps using either the Trusted Registry UI or a YAML file.
 
-While there is a default storage backend, `filesystem`, the Trusted Registry offers other options that are cloud-based. This flexibility to configure to a different storage backend allows you to:
+The default storage backend, `filesystem`, stores and serves images from the
+*local* filesystem.  In a HA setup this fails, as each node can only access its
+own files.
+
+DTR allows you to confiugure your image storage via distributed stores, such as
+Amazon S3, NFS, or Google Cloud Storage. This flexibility to configure to a
+different storage backend allows you to:
 
 * Scale your Trusted Registry
 * Leverage storage redundancy
 * Store your images anywhere in the cloud
 * Take advantage of other features that are critical to your organization
 
 At first, you might have explored Docker Trusted Registry and Docker Engine by
```

Comment:
If this article starts with a good intro and the diagram, we can delete these two paragraphs. They have a bunch of words but teach nothing.